### ISSUE TITLE
Keep local affect keep remote number of backups

### ISSUE THREAD

**Issue by pniewiadowski** [thomasmauerer/hassio-addons#195]
When you set `Keep remote` to `7` and `Keep local` to `7` you will end up with `7` backups in both places.

When you set `Keep remote` to `7` and `Keep local` to `3` you will end up with `3` backups in both places. I think, expected is to have `3` local and `7` remote.

**Comment by glemag** [comment]
I have the issue like this:

**Comment by glemag** [comment]
I have an issue like this: 

1. `Keep local` set to 5
2. `Keep remote` set to 10
3. The quantity of remote backup saving just last 5 backups

**Comment by glemag** [comment]
Any updates on this, is anybody hav a problem which I've mentioned above?

**Comment by pniewiadowski** [comment]
I've stopped using this since we have more enhanced backups in Home Assistant itself.

**Comment by glemag** [comment]
> I've stopped using this since we have more enhanced backups in Home Assistant itself.

Could you please share where in enchanced backup I can setup how many backups I can store local and how many backups I can store on remote? Ex. 7 local / 14 remote. 

Do I need to set 2 different schedules for local and remote? I don't see this option there.

**Comment by pniewiadowski** [comment]
I don't think you can right now, but I give up on this since the native backup would be probably the first option to go. I expect it, to have the feature in upcoming releases.

**Comment by bcutter** [comment]
Oh dear... I only discovered this issue now, wondering why I have so significantly less backups stored remotely than configured.
- `local=45`
- `remote=90`

But remote=local=45 for both locations. That's a no-go.

Will you be able to fix this @thomasmauerer ?
